generatorreceiver/topos/hipster_shop.yaml

```diff
@@ -82,30 +168,36 @@ topology:
                 starter: bulbasaur
         /cart:
           downstreamCalls:
-            cartservice: /GetCart
-            recommendationservice: /GetRecommendations
-          latencyConfigs:
-            - p0: 25ms
-              p50: 75ms
-              p95: 100ms
-              p99: 120ms
-              p99.9: 150ms
-              p100: 200ms
+            - service: cartservice
+              route: /GetCart
+            - service: recommendationservice
+              route: /GetRecommendations
+          latencyPercentiles:
```

Comment:
This `latencyPercentiles` section was also changed in main - a rebase should correct it.